### ISSUE TITLE
Time format

### ISSUE THREAD

**Issue by SvenJuergens** [jweiland-net/events2#115]
hi,
it seems that at the moment you can only use the time format hh:mm. It would be nice if you could change this yourself. eg "10 - 11 Uhr" or 10.15 - 13.45 Uhr"

**Comment by froemken** [comment]
Hello @SvenJuergens 

multiple solutions:
1.) Use replace VH from vhs extension https://fluidtypo3.org/viewhelpers/vhs/2.1.3/Format/ReplaceViewHelper.html and replace ":" with "."
2.) Use explode VH from vhs extensions https://fluidtypo3.org/viewhelpers/vhs/2.1.3/Iterator/ExplodeViewHelper.html and split hour and minute into pieces
3.) Or, if you want, I can build a HourMinuteSplitterViewHelper or so, where you can set divider on your own.

What's your favorite?

Stefan

**Comment by SvenJuergens** [comment]
hello @froemken 

first thought of an ViewHelper too, but then I remembered typoscript can that too :) here my current WorkAround.

but in general a kind of DateTime object would be nice

```
<f:if condition="{time.timeBegin} && {time.timeEnd}" >
    <f:comment> 
       <!-- time format: 11 - 17 Uhr oder 11 - 17.45 Uhr -->
   </f:comment>
       <f:cObject typoscriptObjectPath="lib.eventReplacement">{time.timeBegin}</f:cObject> – <f:cObject typoscriptObjectPath="lib.eventReplacement">{time.timeEnd}</f:cObject> Uhr
</f:if>

<f:if condition="{time.timeBegin} && {time.timeEnd} == ''" >
    <f:comment>
      <!-- time format: 10.00 Uhr oder 18.30 Uhr -->
   </f:comment>
    <f:cObject typoscriptObjectPath="lib.eventReplacementOnlyDots">{time.timeBegin}</f:cObject> Uhr
</f:if>
```

```
lib.eventReplacementOnlyDots = TEXT
lib.eventReplacementOnlyDots{
    current = 1
    replacement{
        10 {
            search = :
            replace = .
        }
    }
}
lib.eventReplacement = TEXT
lib.eventReplacement{
  current = 1
   replacement{
   	10 {
     	search = :00
     	replace = 
   	}
    20{
        search = :
     	replace = .
    }  
  }
}

```

**Comment by froemken** [comment]
Hello @SvenJuergens 

I have just released version 3.7.0. (currently not TER)
With this new release there are 3 new methods in time object
time.timeEntryAsDateTime
time.timeBeginAsDateTime
time.timeEndAsDateTime

Use them as following:
{time.timeEntryAsDateTime->f:format.date(format: 'H.i')}

These methods will only be filled, if a time was set like 12:30 or 00:00. Only if value is an empty string these methods will return NULL which is good catchable through Fluid f:if

Stefan
